diff --git a/lib/git.js b/lib/git.js
index 267ef1d7a38a4de2c1b9986333eb8b2782ec358b..e807eceaf87e61276117d2970eec010ae3c754e2 100644
--- a/lib/git.js
+++ b/lib/git.js
@@ -35,7 +35,7 @@ async function add(files, execaOptions) {
  * @throws {Error} if the commit failed.
  */
 async function commit(message, execaOptions) {
-  await execa('git', ['commit', '-m', message], execaOptions);
+  await execa('git', ['commit', '--cleanup=verbatim', '-m', message], execaOptions);
 }
 
 /**
